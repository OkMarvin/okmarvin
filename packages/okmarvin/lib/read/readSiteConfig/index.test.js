const path = require('path')
const readSiteConfig = require('./index')
test('readSiteConfig', async () => {
  const conn = {
    root: path.join(__dirname, '..', 'fixtures')
  }
  const cb = jest.fn((_err, data) => {
    expect(data).toMatchSnapshot()
  })
  await readSiteConfig(conn, cb)
})
