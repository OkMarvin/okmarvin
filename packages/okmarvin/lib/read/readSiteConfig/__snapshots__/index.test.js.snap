// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`readSiteConfig 1`] = `
Object {
  "author": "陈三",
  "description": "记录人、事与代码",
  "favicon": "",
  "lang": "zh",
  "layoutHierarchy": Object {
    "404.js": Array [
      "404.js",
      "index.js",
    ],
    "category.js": Array [
      "category.js",
      "archive.js",
      "index.js",
    ],
    "index.js": Array [
      "index.js",
    ],
    "page.js": Array [
      "page.js",
      "singular.js",
      "index.js",
    ],
    "post.js": Array [
      "post.js",
      "single.js",
      "singular.js",
      "index.js",
    ],
    "tag.js": Array [
      "tag.js",
      "archive.js",
      "index.js",
    ],
  },
  "permalink": "/:dir/:filename",
  "rss": true,
  "theme": "./themes/hello",
  "themeColor": "",
  "title": "陈三",
  "toc": true,
  "url": "https://blog.zfanw.com",
}
`;
